CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+Guide to new contributors
+=========================
+
+Thanks for contributing to `ingraind` and RedBPF!
+
+Before getting into the best way to participate in the project, please
+take a look at our [code of conduct](./CODE_OF_CONDUCT.md).
+
+## Have a question?
+
+If you have a question about using
+[ingraind](https://github.com/ingraind/ingraind) or
+[RedBPF](https://github.com/ingraind/redbpf), please raise a ticket in
+the respective repository, or visit our [Matrix
+channel](https://app.element.io/#/room/!vCJcBZDeGUXaqSvPpL:rustch.at?via=rustch.at).
+
+## Would like to contribute something?
+
+If you have an exact requirement with a solution, and `ingraind` or
+RedBPF do not support it yet, feel free to open a PR.
+
+We aim to maintain a high quality of contributions, and therefore all
+PRs need to go through a review process. During the review period you may be
+asked to make changes to the code.
+
+In case you are not certain that your code quality or the feature
+you're implementing is suitable for the project, please open an issue
+with the question, or send the PR anyway. This allows the maintainers
+to give you hands on feedback, and highlight anything you may not have
+considered.
+
+A quick checklist of things a PR should have:
```

Comment:
is this checklist ordered by importance of the items?

All items are mandatory? It would be better to separate the must-do and preferred guidelines.
I hope that this guideline does not put much pressure to the potential contributors haha :smile:

Reply:
Good point. I don't consider any of this mandatory, only "nice to have". Might be worth clarifying :)